### PR TITLE
Make sure image, namespace, and tag match

### PR DESCRIPTION
**Pull request by liggitt**
Fixes a problem found trying out the Jenkins example. The build the Jenkins job triggers outputs to a "test" tag, which triggers a new test deployment. After tests run, the image gets tagged to "prod" as an independent step, which triggers a new prod deployment.

While the test build is running, the UI should not show a message in the prod deployment stream about a new deployment happening automatically when the build completes.

@jwforres review

### CODE REVIEW

**Comment by liggitt** [comment]
Corrected appearance ("build running") only appears under the test service/deployment stream
20150304_170517_builds

**Comment by liggitt** [comment]
According to API docs, if namespace is empty in a deployment trigger `from`, it defaults to the namespace of the deployment config. If namespace is empty in a build `to`, it defaults to the namespace of the build.

**Comment by liggitt** [comment]
Also tweaked pod template styles to tighten up whitespace:
Before:
20150304_173923_before

After:
20150304_173948_after

File: assets/app/scripts/controllers/overview.js [hunk 1]
```diff
@@ -241,20 +241,37 @@ angular.module('openshiftConsole')
 
 
     var associateDeploymentConfigTriggersToBuild = function(deploymentConfig, build) {
+      // Make sure we have both a deploymentConfig and a build
       if (!deploymentConfig || !build) {
         return;
       }
+      // Make sure we have a build output
+      if (!build.parameters.output.to) {
```
Comment (jwforres) [inline assets/app/scripts/controllers/overview.js]: build.parameters.output.dockerImageReference is also a possibility

@ironcladlou do deployment config image triggers only trigger off of image repositories?  or can they trigger off of a docker registry directly
@bparees if a build's output refers to dockerImageReference then that is not an image repository correct, it pushes directly to a docker registry?

**Comment by jwforres** [comment]
otherwise LGTM

**Comment by bparees** [comment]
@jwforres correct.

**Comment by bparees** [comment]
@jwforres but of course if that dockerimagereference is also represented by an imagerepo that happens to point to it, the imagerepo will get updated and potentially cause downstream effects.

**Comment by liggitt** [comment]
[merge]

**Pull request merged**